evaluation/ragacli_pyproject/pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "raga_once"
+version = "0.0.1"
+authors = [
+    { name = "Kielo", email = "lanture1064@gmail.com" },
+]
+description = "A one-step Ragas cli tool to evaluate RAG apps"
+readme = "README.md"
+requires-python = ">=3.7"
+classifiers = [
```

Comment:
can we add some dependencies like `ragas,langchain`?

Reply:
yes and I'm now figuring out how to add them 😂

Reply:
> pypi managed requirement automatically.